common/lib/xmodule/xmodule/static_content.py

```diff
@@ -81,7 +90,7 @@ def _write_styles(selector, output_root, classes):
 
     css_fragments = defaultdict(set)
     for class_ in classes:
-        class_css = class_.get_css()
+        class_css = getattr(class_, css_attribute)()
         for filetype in ('sass', 'scss', 'css'):
             for idx, fragment in enumerate(class_css.get(filetype, [])):
                 css_fragments[idx, filetype, fragment].add(class_.__name__)
```

Comment:
For XBlocks, the name would change but since the styles are scoped with view names like the following (in `/common/static/xmodule/descriptor|modules/css/_module-styles.scss`) everything should continue to work correctly:
```
.xmodule_edit.xmodule_ProblemBlock {
}
```